### DISCUSSION TITLE
Welcome to nuzlocke-generator Discussions!

### DISCUSSION THREAD

**Discussion by emzinnia** [EmmaRamirez/nuzlocke-generator#452] [General]
## 👋 Welcome!
Feel free to...
  * Ask questions about how to use the nuzlocke-generator
  * Propose features that you would like input on
  * Vent about your nuzlocke

**Comment by curufinwe879** [comment]
I would like to access the images for the pokemon so I can download them onto my machine, how do I access those in bulk for downloading? I can do one at a time in the team and crop the image if needed but would prefer to have the images themselves if possible

**Reply by emzinnia** [reply]
![bd1795bd-05d1-44cf-b0db-7e8881bbf3be](https://user-images.githubusercontent.com/4407518/141599214-b87bdc74-b842-439c-a363-2934b5793da1.png)

The simplest way is to download the code repo -- the `src/img` folder will contain most pokemon images, `src/assets` contains icons and some images used in the app

**Comment by RonBalaban** [comment]
Hi, I love your generator- been using it for several of my runs for a long time now, and I had a suggestion regarding making the Location Checklist less cluttered. Not sure if something like this is feasible, but would it be possible to filter Met Locations for Genlocke purposes? Say I played two different games, but both of them have a shared location- for example, FireRed and HeartGold both have Kanto Route 2 in them. I caught and recorded a Caterpie on Route 2 in FireRed, and am now playing Heartgold. The Location Checklist shows that I already have an encounter for Route 2, even though I haven't caught anything there in my HeartGold game. 

One way I can see this is by using the 'Game of Origin' (GOO)  section in a Pokemon's profile. If I've selected that I'm playing HeartGold on the top-left drop-down selection where I choose the game, and that Caterpie's GOO was FireRed, then that Caterpie should not show in the Location Checklist section. This tells me that I still have an encounter for that area in my HeartGold run. Essentially, only display recorded Pokemon whose GOO matches the selected Game.    
  
 


Another suggestion I have is regarding something called the Pinwheel Clause- that is when the main area actually consists of multiple smaller areas. For example, in Sun/Moon, Route 1 has multiple separate areas in it, each with separate encounter tables- Route 1 Proper, Trainer's School, and Hau'oli Outskirts [https://pokemondb.net/location/alola-route-1]. The Location Checklist could accommodate this by listing separate sections for each location, like this; 'Route 1 Proper', 'Route 1 Trainer's School', 'Route 1 Hau'oli Outskirts'




Finally, last suggestion. When a player enters the Met Location of a Pokemon, allow the name-catch to work anywhere. So if I encountered a Pokemon on Route 11, don't force me to spell out 'Route', simply allow me to type '11', and let 'Route 11' show as a suggested location. This could probably be done using wild cards in Regex, such as '.* [input_location] .*', but I'm unfamiliar with the code format.

**Comment by zondrethevexed** [comment]
Hey I made another post, but I just had another idea. 

It would be cool if we could import pokemon in the showdown format. It could save time inputting information.

Here's an example from my current nuzlocke.

Diesel (Machop) (M)
Ability: Steadfast
Level: 3
Friendship: 70
Serious Nature
IVs: 22 HP / 12 Atk / 11 Def / 21 SpA / 6 SpD / 9 Spe
- Low Kick
- Leer

It would be cool if I could just paste this in and it would take the pokemon, nickname, gender, ability, level, nature, and moves from this. These are really fast to export from a game via PKHex
